src/components/App/App.js

```diff
@@ -29,6 +41,14 @@ class App extends Component {
 					Please take an item and show it to the camera
 				</header>
 				<hr />
+				Slack Username:
+				<input
+					value={this.state.currentUser}
+					onChange={event => {
```

Comment:
We could extract event handlers to methods, it'd help keep render focused on "here's what I look like" and has some perf benefits but we shouldn't worry about them for now :-)